### ISSUE TITLE
Pass css class in `<ValidationSummary />` to the generated `<ul>`

### ISSUE THREAD

**Issue by ChristopherHaws** [dotnet/aspnetcore#43860]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

I am trying to pass additional css classes to the generated `ul` element. I am applying general formatting via the `validation-errors` css class, but I would like to be able to pass utility classes for spacing (i.e. `pt-2` to add 8px of padding to the top of the validation summary).

### Describe the solution you'd like

Update `<ValidationSummary />` to pass unmatched attributes (via splatting) to the `validation-errors` element. In the case of `class`, it should append the passed in classes to the existing `validation-errors` class.

### Additional context

For example:
```razor
<ValidationSummary class="pt-2" data-test="validation-errors" />
```

should result in:
```html
<ul class="validation-errors pt-2"  data-test="validation-errors">
    <!-- Validation Errors -->
</ul>
```

**Comment by SteveSandersonMS** [comment]
Thanks for the suggestion. I think we'd achieve this by adding a new parameter, `CssClass`, to the `ValidationSummary` component and then render that into the `class` attribute. This is the same pattern used in some of the other components.

We'd take a PR for this, assuming it also contains E2E tests.

**Comment by ChristopherHaws** [comment]
I would be happy to pick up this issue. I do have a question however. From what I can see in the existing components, `CssClass` is usually a "get only" property that combines the `class` attribute from `AdditionalAttributes` using `AttributeUtilities.CombineClassNames`. The only exception I found to this is NavLink, which updates the `CssClass` property `OnParametersSet` to include the `class` attribute from `AdditionalAttributes`.

In other words, from what I can see in the existing components, `class` is always set by passing the `class` attribute via `AdditionalAttributes`. I can certainly add a `CssClass` property and append it to the existing class in the component and that will work, but I wanted to double check since it doesn't seem like that's how any of the other components work.

Thanks!

**Comment by artemsemdev** [comment]
I’d like to take this one.
From the current implementation, it looks like ValidationSummary already accepts unmatched attributes, but class currently overrides validation-errors instead of appending to it.
My plan is to preserve the default validation-errors class, append any user-supplied class value, and cover the behavior with E2E tests.
If you’d still prefer a dedicated CssClass parameter instead, I can follow that direction.
